torch/_inductor/kernel/mm.py

```diff
@@ -696,14 +699,18 @@ def tuned_mm(mat1, mat2, *, layout=None):
     mm_configs = V.choices.get_base_mm_configs(device_type)
     persistent_mm_configs = V.choices.get_persistent_mm_configs(device_type)
     extra_mm_configs = V.choices.get_extra_mm_configs(device_type)
+    lookup_dict = get_lookup_table([mat1, mat2], name)
 
     dtype = mat1.get_dtype()
     if is_nonzero and use_triton_template(layout):
+        # Search if the config is triton
+        valid_config = lookup_template_dict(lookup_dict, "triton")
         for config in mm_configs(
             m,
             n,
             k,
-            **mm_config_kwargs(device_type, _is_large_block_for_cpu, dtype.itemsize),
+            *mm_config_kwargs(device_type, _is_large_block_for_cpu, dtype.itemsize),
+            valid=valid_config,
```

Comment:
Using config filtering here may be error prone.  If we change the config space then you could get zero configs.  Can we lookup the correct config from the table?